nuxt.config.ts

```diff
@@ -104,7 +104,7 @@ export default defineNuxtConfig({
       // our default translation server #76
       translateApi: '',
       defaultServer: 'mozilla.social',
-      singleInstance: false,
+      singleInstance: true,
```

Comment:
Okay!  Front end developers don't necessarily use Docker to fire things up and work on this code.  Setting this to true here shouldn't (as far as I understand it) effect the Docker instance, staging or production ... but it should more effectively match up local client only dev and what's going on everywhere else.